docs/user/appendix/contribution-guidelines.md

```diff
@@ -0,0 +1,224 @@
+
+There are two important questions to answer before adding new content to this book:
+
+- Where to include this content?
+- In which format to present it?
+
+This guide aims to provide context for answering both questions.
+
+## Table of contents
+
+<!-- toc -->
+
+## Where to add new content?
+
+This book is divided in five different sections. Each section contains pages that are of a specific
+type. New content will fit in one of these section. Following is an explainer on what kind of content
+fits in each section.
+
+### Overview
+
+_Is the content you want to add an essay or high level explanation on some aspect of Glean?_
+
+The overview section is the place to provide important higher level context for
+users of Glean. This section may include essays about Glean’s views, principles,
+data practices, etc. It also contains primers on information such as what is the Glean
+SDK and Glean.js.
+
+### User Guides
+
+_Is the content you want to add a general purpose guide on a Glean aspect or feature?_
+
+This section is the place for Glean SDK user guides. Pages under this section
+contain prose format guides on how to include Glean in a project, how to add metrics,
+how to create pings, and so on.
+
+Pages on this section may link to the API Reference pages, but should not
+themselves be API References.
+
+Guides can be quite long, thus we should favour having one page per language binding instead
+of using tabs.
+
+### API Reference
+
+_Is the content you want to add a developer reference on a specific Glean API?_
+
+This section of the book contains reference for all of Glean’s user APIs.
+Its pages follow a strict structure.
+
+Each API description contains:
+
+- A title with the name of the API.
+  - It’s important to use titles, because they automatically generate links to that API.
+- A brief description of what the API does.
+- Tabs with examples for using that API in each language binding.
+  - Even if a certain language binding do not contain a given API, the tab will be included in
+  the tabs bar in the disabled state.
+
+The API Reference pages should not contain any guides in prose format, they should all be linked from the User’s Guide when convenient.
+
+### Language Binding Information
+
+_Is the content you want to add a language binding specific guide on a Glean feature?_
+
+Language bindings may require some dedicated pages, these section contains these pages.
+Each language binding has a top level section under this section, specific pages live
+under these titles.
+
+### Appendix
+
+_Is the content you want to add support content for the rest of the content on book?_
+
+The appendix contains support information related to the Glean SDK or the content of this book.
+
+## In which format to present content?
+
+### General guidelines
+
+#### Link to other internal pages and sections whenever that is possible
+
+Each page of the book should be written as if it were the first page a user is visiting ever.
```

Comment:
do we want to add a little explanation about why? I'm not questioning this, I think it's exactly what we need to do, but maybe explaining why will help our future selves and others reading this later on :)